### DISCUSSION TITLE
Vue 3 and Vuetify 3

### DISCUSSION THREAD

**Discussion by ibnu-ja** [CzBiX/qb-web#152] [Ideas]
Thanks for the UI. It's great.

Anyway, any plan to adopt Vue 3 with Vuetify 3 Beta? I think most of the components here is ready to migrate.

**Comment by CzBiX** [comment]
yes, I do have this plan. rewrite this project with vite + vue 3. but I think we need to give Vuetify a little more time.

**Reply by ibnu-ja** [reply]
what is it that we need to wait?
I think most of component is ready in 3.0 release.
I can help. I think it's better to create new project board.

**Reply by Izumiko** [reply]
Vue 2 has officially reached EOL now. Has there been any further progress on the upgrade plan?
